lib/json_api_client/errors.rb

```diff
@@ -10,28 +11,25 @@ def initialize(env)
     class ClientError < ApiError
     end
 
+    # class ResourceImmutableError < StandardError
+    #   def initialize(msg = 'Resource immutable')
+    #     super msg
+    #   end
+    # end
```

Comment:
the commented out blocks are from the upstream repository.

In the next PR, the `status.rb` class will be fixed accordingly